mssqlcli/mssqlcliclient.py

```diff
@@ -50,6 +50,9 @@ def __init__(self, mssqlcli_options, sql_tools_client, owner_uri=None, **kwargs)
         self.server_edition = None
         self.is_cloud = False
 
+        # non-interactive mode options
+        self.query = mssqlcli_options.query
+
```

Comment:
Can we remove this now that we are passing in the query string via the execute_query method argument?